frontend/src/models/models.html

```diff
@@ -47,7 +47,7 @@
             Create
           </button>
           <button
-            @click="shouldShowFieldModal = true"
+            @click="this.selectedPaths = [{ path: '_id' }]; shouldShowFieldModal = true;"
```

Comment:
This is slightly problematic because if the user selected some paths, I would expect opening up the field selection modal would show the paths I already have selected.

I think it would be better to start with just `_id` _unless_ the user has already made some explicit field selection, or the field selection is pulled from the URL: 
![image](https://github.com/user-attachments/assets/092d01f5-ed5c-4fb7-953d-b2a30b9c7502)